readme.md

```diff
@@ -819,6 +820,92 @@ Type: `string`
 
 The IP address used to send the request from.
 
+### HTTPS
+
+##### ca
```

Comment:
I would prefer to use the full word for this. (I do realize this is just passed through to Node.js, but we can do better).